understanding/22/focus-not-obscured-minimum.html

```diff
@@ -48,14 +53,95 @@ <h2>Intent of Focus Not Obscured (Minimum)</h2>
          <p>Another form of obscuring can occur where light boxes or other semi-opaque effects overlap the item with focus. While less than 100 percent opacity is not causing the component to be <q>entirely hidden</q>, such semi-opaque overlaps may cause a failure of <a href="../understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 1.4.11 should be evaluated (and pass) while the focus indicator is under the semi-opaque component. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
 
     </section>
+    <section id="Movable"><h3>User-movable content</h3>
+      <p>This SC contains a note regarding content that can be repositioned. If users can move content regions, then they can potentially position the movable content such that it obscures other content that may receive focus. In such a case, the author is only responsible for ensuring that the movable content <em>in its initial position</em> does not obscure the item receiving focus.</p>
+      <p>This note is intended to accommodate a common interaction in complex applications such as authoring tools, where the main editing region (also called a canvas) can be enhanced by displaying toolbars or other panels, which can be repositioned around the canvas. It is possible to design such toolbars so they do <em>not</em> obscure focus. Authors are encouraged to do so, as well as pursue techniques which ensure equitable keyboard use of such toolbars. However, in recognition of the complexities involved in responsive design as well as in supporting the ability to transform the text size and spacing of content, only the starting position of such movable panels is assessed.</p>
+
+      <figure id="">
+         <img src="" alt="" />
+         <figcaption>Placeholder for future simplified illustration showing an abstract layout of an authoring tool UI with a user-movable toolbar</figcaption>
+     </figure>
+  
+    </section>
+    <section id="Disclosed"><h3>User-disclosed content</h3>
+   <p>This SC contains a note regarding content that is opened or disclosed by the user. One example of such content is a menu button opened by a user that discloses a list of choices over pre-existing content on the screen. Most such disclosed content can obscure other information on the screen, but it does not obscure an item receiving keyboard focus, because the new content doesn't persist through a change of focus. However, authors may create user-opened content that is intentionally designed to persist until closed by the user, such as a chat window. Such persistent content has the potential to fail Focus Not Obscured (Minimum). Various types are described in this section. All can be designed so that they pass this Success Criterion.</p>
```

Comment:
```suggestion
   <p>This SC contains a note regarding content that is opened or disclosed by the user. One example of such content is a menu button opened by a user that discloses a list of choices over pre-existing content on the screen. Such disclosed content can obscure other information on the screen, but it does not obscure an item receiving keyboard focus, because the new content doesn't persist through a change of focus. However, authors may create user-opened content that is intentionally designed to persist until closed by the user, such as a chat window. Such persistent content has the potential to fail Focus Not Obscured (Minimum). Various types are described in this section. All can be designed so that they pass this Success Criterion.</p>
```